src/openforms/utils/tests/test_decorators.py

```diff
@@ -0,0 +1,72 @@
+from django import forms
+from django.contrib.admin.widgets import AdminTextInputWidget
+from django.db import models
+from django.test import SimpleTestCase
+from django.test.client import RequestFactory
+
+import requests
+
+from openforms.utils.decorators import supress_requests_errors
+
+
+class DbFieldsExceptionHandlerTests(SimpleTestCase):
+    def setUp(self):
+        self.db_field = models.CharField(
+            "db field",
+            blank=True,
+            help_text="help text",
+        )
+
+        self.request = RequestFactory().get("/test")
+
+    def test_no_exception_detected(self):
+        @supress_requests_errors(fields=["db field"])
+        class TestClass:
+            formfield_overrides = {models.CharField: {"widget": AdminTextInputWidget}}
+
+            def formfield_for_dbfield(self, db_field, request, **kwargs):
+                return forms.CharField(
+                    label="db field",
+                    help_text="help text",
+                )
+
+        form_field = TestClass.formfield_for_dbfield(
+            TestClass, db_field=self.db_field, request=self.request
+        )
+
+        self.assertIsInstance(form_field, forms.CharField)
+        self.assertNotIn(
+            "Could not load data - enable and check the request logs for more details",
+            form_field.widget.render("db field", None),
+        )
```

Comment:
I don't know how I feel about this artificially constructed classes that pretend to be an admin class.

Can't we create real model admin classes and test those?

Something like:

```py
from django.contrib.admin import AdminSite
test_site = AdminSite(name="test")

@suppress_requests_errors(fields=["some_field"])
class TestAdmin(admin.ModelAdmin):
    pass

# bind a model to it
model_admin = TestAdmin(SomeModel, test_site)

# now you can test this using:
formfield = model_admin.formfield_for_db_field(Model._meta.get_field("some_field"), ...)
```

This would at least accurately describe the intended usage.